xpasses/XS002/testdata/src/a/main.go

```diff
@@ -0,0 +1,66 @@
+package a
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+const myConstKey = "my_const_key"
+
+func f() {
+	_ = schema.Resource{
+		Schema: map[string]*schema.Schema{ // want "schema attributes should be in alphabetical order"
+			"name": {},
+			"arn":  {},
+		},
+	}
+
+	_ = schema.Resource{
+		Schema: map[string]*schema.Schema{ // want "schema attributes should be in alphabetical order"
+			"name": {},
+			"arn":  {},
+			"block": {},
+		},
+	}
+
+	_ = schema.Resource{
+		Schema: map[string]*schema.Schema{ // want "schema attributes should be in alphabetical order"
+			"block": {},
+			"arn":  {},
+			"name": {},
+		},
+	}
+
+	_ = schema.Resource{
+		Schema: map[string]*schema.Schema{ // want "schema attributes should be in alphabetical order"
+			"arn":  {},
+			"name": {},
+			"block": {},
+		},
+	}
+
+	_ = schema.Resource{
+		Schema: map[string]*schema.Schema{ // want "schema attributes should be in alphabetical order"
+			"arn": {},
+			"name":  {},
+			myConstKey: {},
+		},
+	}
+
+	var myVarKey = "my_var_key"
+	_ = schema.Resource{
+		Schema: map[string]*schema.Schema{ // want "schema attributes should be in alphabetical order"
+			"arn": {},
+			"name":  {},
+			myVarKey: {},
+		},
+	}
+
+	myAssignKey := "my_assign_key"
+	_ = schema.Resource{
+		Schema: map[string]*schema.Schema{ // want "schema attributes should be in alphabetical order"
```

Comment:
Given the change above should not panic in this case, but also pass since the literal string keys are sorted. 👍 

```suggestion
		Schema: map[string]*schema.Schema{
```